data/remnant/remnant missions.txt

```diff
@@ -3466,6 +3466,45 @@ event "remnant: cognizance timer"
 
 
 
+mission "Remnant: Cognizance 19"
+	name "Deliver nav data"
+	description "Deliver navigational data on the new systems to <destination>."
+	landing
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Cognizance 18: done"
+	on offer
+		conversation
+			`As you finish your landing checks your comlink notifies you that you have an incoming message from Dawn. It promptly displays a short message:`
+			`	"Greetings, Captain <first>! We are starting to compile updated navigational data for the new systems, and would like a copy of the data your nav system recorded. Stop by <planet> and the starport automated systems will get a copy of the data during the systems check. It is a pretty small data set, so there should be no need to use data crystals. Thank you!"`
```

Comment:
```suggestion
			`	"Greetings, Captain <first>! We will be compiling updated navigational charts for the new systems and would like a copy of the data your nav system recorded. Stop by <planet> and the starport systems will copy the data during the systems check. It's a pretty small data set, so there's no need to use data crystals. Thank you!"`
```

The logical question a player might ask is, "why didn't those starport systems get the data from me when I landed here?", especially since the player may land on any remnant world to trigger this.